core/View.php

```diff
@@ -287,6 +287,9 @@ public function render()
         // don't send Referer-Header for outgoing links
         if (!empty($this->useStrictReferrerPolicy)) {
             Common::sendHeader('Referrer-Policy: same-origin');
+        } else {
+            // always send explicit default header
+            Common::sendHeader('Referrer-Policy: no-referrer-when-downgrade');
```

Comment:
@Findus23 do you have maybe some thoughts on this?

Reply:
`no-referrer-when-downgrade` is the default when no header is sent, so sending it when we normally wouldn't send anything shouldn't change anything.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referrer-Policy

Reply:
thanks @Findus23 be good if someone could still test the PR and have a look at tests. Not sure if I find time today.

Reply:
Aye it's the (implicit) default; this PR caters to our setup where we set a strict policy if nothing is explicitly set by the application (see [this comment](https://github.com/matomo-org/matomo/issues/14764#issuecomment-595180297)). Just in case you want to cater for our way of thinking (not sure if we're the outlier here or if more people might want to set their policies with `setifempty`).